kotlin/src/main/java/app/rive/runtime/kotlin/RiveArtboardRenderer.kt

```diff
@@ -429,11 +427,10 @@ open class RiveArtboardRenderer(
                 }
 
             }
-
         } else {
             this.activeArtboard?.advance(0f)
+            start()
```

Comment:
The other branch of this `if` has a few `play()` calls that internally call `start()` already. This is just to prevent it being called multiple times unnecessarily!